docs/prms/oi1.rst

```diff
@@ -1,4 +1,6 @@
 Omics Integrator I
 ==================
 
-Here's a description of the PRM.
+Omics Integrator I is a pathway reconstruction algorithm. See the source code:
+https://github.com/fraenkel-lab/OmicsIntegrator (licensed under BSD-2),
```

Comment:
Here is the information on the OI1 parameters

https://github.com/fraenkel-lab/OmicsIntegrator#:~:text=w%20%3D%20float%2C%20controls,optimization%0A%20%20%20%20%20%20%20%20%20%20%20%20(default%201)

https://github.com/fraenkel-lab/OmicsIntegrator#:~:text=Usage%3A%20forest.py,seed.%20Default%20%3D%20None.